components/arm/arm.go

```diff
@@ -65,8 +65,7 @@ type Arm interface {
 	resource.Resource
 	referenceframe.ModelFramer
 	resource.Shaped
-	resource.Actuator
```

Comment:
I'm sorry - I'd rather not have the actuator interface be nested in Input enabled in some resources (arm, gantry, kinematic base) and not in others (motor, servo, base). Could we keep it flat?

Reply:
@raybjork Do you want to weigh in on this?

Reply:
Is this objection from a readability perspective Rand?

Reply:
@randhid and I talked offline and agreed on a compromise solution, renaming this interface to `InputEnabledActuator` I want to tag @biotinker on this too since he might have thoughts

Reply:
The addition of `resource.InputEnabledActuator` as a wrapper seems weird to me.

I think it would be cleaner to move `referenceframe.InputEnabled` to `resource`, and then simply keep it separate from `Actuator`.

Reply:
This is coming up because we want to be able to more easily reference things we move in the motion service. Right now we store them as input enabled and have to assert they are actuators, but if we had one type that fully encompasses things we can move that would be cleaner

Reply:
That makes sense.
Another option is to put this interface privately on the motion service itself. It can cast everything to its own internal type then, once.

Also OK with this as-is but it seems somewhat clunky.

Reply:
that actually seems like the best option, thanks for the suggestion peter!